tests/cases/docker/chrome-devtools-frontend-next/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:current
+RUN git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git /depot_tools
+ENV PATH=/depot_tools:$PATH
+WORKDIR /
+RUN mkdir devtools
+WORKDIR devtools
+RUN fetch devtools-frontend
+WORKDIR devtools-frontend
+RUN gn gen out/Default
+COPY --from=typescript/typescript /typescript/typescript-*.tgz typescript.tgz
+RUN mkdir /typescript
+RUN tar -xzvf /typescript.tgz -C /typescript
+RUN ln -s /typescript/package ./node_modules/typescript
+RUN autoninja -C out/Default front_end
```

Comment:
For some reason if this last step is a `CMD` then it doesn't do anything at all. I am not sure why, but the `RUN` works as expected.

Reply:
A docker file is split into two phases, build and run. All the `RUN`s build the FS in the `Dockerfile`. The `CMD` is what is executed (by default) when you actually _execute_ the built docker image. We use that here - only the `CMD` output is baseline'd.

Reply:
Ah I see, thanks for the explanation.